docs/components/modeler/bpmn/script-tasks/script-tasks.md

```diff
@@ -10,26 +10,52 @@ JavaScript, or Python.
 
 ![task](assets/script-task.png)
 
-Script tasks behave exactly like [service tasks](/components/modeler/bpmn/service-tasks/service-tasks.md). Both task
-types are based on jobs and [job workers](/components/concepts/job-workers.md). The
+:::info
+Camunda Platform 8 supports alternative task implementations for the script task. If you want to use your own
+implementation for a script task, see the [job worker implementation](#job-worker-implementation) section below. The
+sections before this job worker implementation apply to the [FEEL expression](/components/modeler/feel/language-guide/feel-expressions-introduction.md)
+implementation only.
+:::
+
+When the process instance arrives at a script task, the integrated [FEEL Scala](https://github.com/camunda/feel-scala)
+engine evaluates the script task FEEL expression. Once the FEEL expression is evaluated successfully, the process
+instance continues.
+
+If the FEEL expression evaluation is unsuccessful, an [incident](/components/concepts/incidents.md) is
+raised at the script task. When the incident is resolved, the script task is evaluated again.
+
+## Defining a script task
```

Comment:
🔧 It might be nice to also provide a section for the variables mapping like we do for the business rule tasks https://docs.camunda.io/docs/next/components/modeler/bpmn/business-rule-tasks/#variable-mappings

We do this for a lot of the other tasks as well, but we're not too consistent it seems.

Reply:
That's a good suggestion @remcowesterhoud, thanks!

I already added the `variable-mappings` section. Have a look.